test_cases/admins_as_venues.json

```diff
@@ -0,0 +1,325 @@
+{
+  "name": "Single-token admin areas analysis should search for venues",
```

Comment:
What would you think about changing the name of the test file and the name of the test suite to match more closely? Right now it shows up near the top of the acceptance tests because the file name starts with `a`, but the name is this big long string, which is a bit unexpected.

Reply:
You can change the `name` property to `description` and add `name` to match the filename.